StarRating/StarRating/ViewController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ViewController.swift
+//  StarRating
+//
+//  Created by Jeffrey Santana on 8/13/19.
+//  Copyright © 2019 Lambda. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		// Do any additional setup after loading the view.
+	}
+
+	@IBAction func updateRating(_ sender: StarControl) {
+		title = "User rating: \(sender.rating) \(sender.rating == 1 ? "star" : "stars")"
+	}
+	
+}
+
+extension UIView {
+	// "Flare view" animation sequence
+	func performFlare() {
```

Comment:
this code would be better to have in the StarControl file, probably with a file private access level.